python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -540,6 +542,87 @@ def run_ipex_fp16_gpu(repo_id,
     torch.xpu.empty_cache()
     return result
 
+def run_deepspeed_transformer_int4_cpu(repo_id,
+                         local_model_hub,
+                         in_out_pairs,
+                         warm_up,
+                         num_trials,
+                         num_beams):
```

Comment:
add a parameter `low_bit`

Reply:
It is found that only limited combinations of low-bit optimization and deepspeed dtype work, e.g. sym_int4 and torch.float16, and others may encounter precision problems, e.g. wrong output encoding with sym_int8 and torch.float16